doc/coq.rst

```diff
@@ -16,8 +16,13 @@ and ``.mlg`` file preprocessing.
 A *Coq theory* is a collection of ``.v`` files that define Coq modules whose
 names share a common prefix. The module names reflect the directory hierarchy.
 
-A *Coq plugin* is an OCaml :ref:`library` that Coq can load dynamically at
-runtime. Plugins are typically linked with the Coq OCaml API.
+A *Coq plugin* is an OCaml :ref:`library` that Coq can load
```

Comment:
Why did you wrap this tighter?

Reply:
Emacs decided to do it.

Reply:
Can you make it consistent with the rest of the file? Emacs should let you modify the fill-column variable to 80.